documentation/configure.markdown

```diff
@@ -881,3 +880,17 @@ default this is `127.0.0.1` only. As this is an insecure channel this
 is the only recommended setting for production environments. 
 
 If you wish to listen on all interfaces, you can specify `0.0.0.0`, for example, although this is generally not recommended for production. 
+
+`[developer]` settings
+-----
+
+The `[developer]` section contains configuration items that may be useful to
+users developing against the PuppetDB API. These should be used with care in
+production, as they may impede performance.
+
```

Comment:
Wondered if there's any reason we might want to be a little more conservative here to start, i.e. possibly along the "only use if you know what you're doing" lines.   I suppose it depends on what we think we might do with it in the future.